src/subdomains/supporting/address-pool/deposit/deposit.service.ts

```diff
@@ -105,18 +113,18 @@ export class DepositService {
       addresses.push(deposit.address);
     }
 
-    addresses.push(this.createPaymentAddress(0));
+    addresses.push(...this.createPaymentAddresses());
 
     for (const chain of applicableChains) {
       await this.alchemyWebhookService.createAddressWebhook({ blockchain: chain, addresses: addresses });
     }
   }
 
-  private createPaymentAddress(accountIndex: number): string {
-    return EvmUtil.createWallet({ seed: Config.payment.evmSeed, index: accountIndex }).address;
+  private createPaymentAddresses(): string[] {
+    return [EvmUtil.createWallet({ seed: Config.payment.evmSeed, index: 0 }).address, Config.payment.moneroAddress];
```

Comment:
I don't think that's quite right. When creating the webhook for Alchemy, no Monero address should be used.